build/ci_utils/src/program/command.rs

```diff
@@ -469,7 +469,22 @@ pub fn spawn_log_processor(
                 match String::from_utf8(line_bytes) {
                     Ok(line) => {
                         let line = line.trim_end_matches('\r');
-                        info!("{prefix} {line}");
+                        let mut command = false;
+                        if let Some(command_at) = line.find("::") {
+                            if let Some(group_at) = line.find("group") {
+                                // we support:
+                                // ::group
+                                // ::endgroup
+                                if command_at < group_at && group_at < command_at + 10 {
+                                    let line_without_prefix = &line[command_at..];
+                                    println!("{line_without_prefix}");
```

Comment:
cleanup

Reply:
Ok, maybe there is a comment needed that `println` is used to print the `::group` mark without the prefix

Reply:
done in 4c0ddd51efb60df3b74c76c5ca8de70e92c18509